website/docs/r/acm_certificate.html.markdown

```diff
@@ -105,8 +105,8 @@ Supported nested arguments for the `options` configuration block:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The ARN of the certificate
-* `arn` - The ARN of the certificate
+* `id` - The ID of the certificate
+* `arn` - The ARN of the certificate (same as `id`)
```

Comment:
We typically try to denote that `id` attribute is the ARN when it is an ARN. It can be confusing to say "ID" in this context because that could mean just the part after `certificate/...` in the ARN.

For additional context, we typically add the `arn` attribute by convention, even if the resource ID is the ARN, since folks seem to expect that attribute name.

See also: https://github.com/terraform-providers/terraform-provider-aws/blob/941de448465dd3619fbde6ced64ab4111b1d5fe2/aws/resource_aws_acm_certificate.go#L217

```suggestion
* `id` - The ARN of the certificate
* `arn` - The ARN of the certificate
```